src/main/java/by/custom/utilcalculator/service/MessageRouter.java

```diff
@@ -1,22 +1,25 @@
 package by.custom.utilcalculator.service;
```

Comment:
Please, move it under `controller` package
```suggestion
package by.custom.utilcalculator.controller;
```

Reply:
Done.